target_hubspot_v4/unified.py

```diff
@@ -205,11 +205,9 @@ def process_contacts(self, record):
             row.update({"id": record.get("id")})
 
         if "id" not in row and row["properties"].get("email"):
-            contact_search = search(dict(self.config), row["properties"].get("email"))
+            contact_search = search_contact_by_email(dict(self.config), row["properties"].get("email"))
             if contact_search:
-                if len(contact_search) > 0:
-                    if contact_search[0].get("id"):
-                        row.update({"id": contact_search[0]["id"]})
+                row.update({"id": contact_search.get("id")})
```

Comment:
Now won't this create cases where have `"id": None`? I wanted to just move the check into the `if` on line `209` not delete it